AGENTS.md

```
@@ -1,11 +1,304 @@
+
+---
+
+# Pull Requests
+
+## Branch Naming
+
+Branch names must include the **affected apps**.
+
+Example:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)